python/examples/langchain-smart-wallet-chatbot/chatbot.py

```diff
@@ -22,34 +22,32 @@
 from langgraph.checkpoint.memory import MemorySaver
 from langgraph.prebuilt import create_react_agent
 
-wallet_data_file = "wallet_data.txt"
-
 load_dotenv()
 
-
 def initialize_agent():
     """Initialize the agent with SmartWalletProvider."""
     llm = ChatOpenAI(model="gpt-4o-mini")
 
+    network_id = os.getenv("NETWORK_ID", "base-sepolia")
+    wallet_data_file = f"wallet_data_{network_id.replace('-', '_')}.txt"
```

Comment:
I wanted to improve the smart wallet examples before I copied their code into other examples.

This improvement is regarding switching networks. Now, rather than a `wallet_data.txt` file that cannot change networks, each network will generate a different wallet file, e.g. `wallet_data_base_sepolia.txt` and `wallet_data_base_mainnet.txt`